<launch>

<arg name="loggerPath"/>


<node pkg="state_controller" name="state_controller" type="state_controller" output="screen"/>

<node pkg="gnss_dummy" name="gnss_dummy" type="gnss_dummy" output="screen"/>

<node pkg="pilot" name="pilot" type="pilot" output="screen"/>
<node pkg="goal_planner" name="goal_planner" type="goal_planner" output="screen"/>


</launch>
